src/fragments/lib/auth/js/social.mdx

```diff
@@ -4,6 +4,10 @@
 
 OAuth support in Amplify uses Cognito User Pools and supports federation with social providers, which will automatically create a corresponding user in the User Pool after a login. [OIDC](https://en.wikipedia.org/wiki/OpenID_Connect) tokens are available in the app after the application has completed this process.
 
+import trigger from "/src/fragments/cli/lambda_triggers_callout.mdx"
+
+<Fragments fragments={{all: trigger}} />
+
```

Comment:
Do we not need this callout for other frameworks?

Reply:
added it for other frameworks as well in the fragment